calcom/cal.com
# feat: Availability slider for orgs

Implements a WTB type of display for users availability and timezones for use in ORGS only

How to test: 
Create a ORG
Invite a few users
go through onboarding with them and change the timezone of each user.
Visit /availability you should see a tab to switch to your team or visit /availability?type=team directly.
Compare generated hours with the worldtimebuddy equivalent. 
Ensure availability matches up with the users default schedule 



https://www.loom.com/share/e0508a3b048e4f4fb4a67682f4373ec1
<img width="1239" alt="CleanShot 2023-08-14 at 14 25 52@2x" src="https://github.com/calcom/cal.com/assets/55134778/b502d418-c04d-41dc-a58d-b2b7c4fd8dfb">
<img width="1243" alt="CleanShot 2023-08-14 at 14 25 49@2x" src="https://github.com/calcom/cal.com/assets/55134778/7f08c6ca-b16b-4076-9bfd-aa6a8db99da8">
